### DISCUSSION TITLE
Filter: Support double asterisk also for e.g. Source

### DISCUSSION THREAD

**Discussion by distantnative** [plausible/analytics#1770] [Anything Else]
Would be great to have wildcard `**` not just for pages/paths, but also in others, e.g. source (example: filtering all sources `masterstud**` for masterstudies.com, masterstudies.de, masterstudy.de).

**Comment by distantnative** [comment]
Ha, sorry - totally overlooked the discussions tab for feature requests. Thanks for moving 🙏

**Reply by metmarkosaric** [reply]
np! thanks for the feedback!
